config/stylus-config.js

```diff
@@ -0,0 +1,33 @@
+const miniCSSExtractPlugin = require('mini-css-extract-plugin')
+const path = require('path')
+
+module.exports = {
+  test: /\.styl$/,
+  include: /src/,
+  use: [
+    process.env.production ? miniCSSExtractPlugin.loader : 'style-loader',
+    {
+      loader: 'css-loader',
+      options: {
+        modules: true,
+        localIdentName: '[name]__[hash:base64:8]',
+        importLoaders: 1
+      },
+    },
+    'postcss-loader',
+    { 
+      loader: 'stylus-loader', 
+      // options: {
```

Comment:
:scissors:

Reply:
This is related to the issue  #1. 
 It'Is gonna be temporary removed on v.1.1 (https://github.com/mariobrusarosco/webpack-boilerplate/releases/tag/v1.1.0) while the issue is active!

Reply:
This is related to Issue #1. It's gonna be temporary removed on  Build's [v1.1.0](https://github.com/mariobrusarosco/webpack-boilerplate/releases/tag/v1.1.0)